### DISCUSSION TITLE
[Support]: Zone within a zone

### DISCUSSION THREAD

**Discussion by ronbruins** [blakeblackshear/frigate#10863] [General Support] [answered]
### Describe the problem you are having

Is it OK to have a zone within a zone. e.g. 
- zone: backyard - covers the whole yard
- zone: loungeset - a square inside the yard zone.

What would be the drawbacks of this?

So alternatively: Would it be better to exclude the 'loungeset' from the yard zone so they each cover their own area?

### Version

NA

### Frigate config file

```yaml
NA
```

### Relevant log output

```shell
NA
```

### FFprobe output from your camera

```shell
NA
```

### Frigate stats

```json
NA
```

### Operating system

Other Linux

### Install method

Docker Compose

### Coral version

USB

### Network connection

Wired

### Camera make and model

reolink

**Comment by NickM-27** [comment, marked as answer]
Yes, there are no issues with that
